media/server/main/source/SharedMemoryBuffer.cpp

```diff
@@ -255,18 +257,26 @@ bool SharedMemoryBuffer::clearData(MediaPlaybackType playbackType, int id, const
                                 common::convertMediaSourceType(mediaSourceType), toString(playbackType), id);
         return false;
     }
+
+    if (MediaSourceType::VIDEO == mediaSourceType)
+    {
+        std::uint8_t *videoData = partitionDataPtr;
+        memset(videoData, 0x00, partition->dataBufferVideoLen);
+        return true;
+    }
     if (MediaSourceType::AUDIO == mediaSourceType)
     {
         std::uint8_t *audioData = partitionDataPtr + partition->dataBufferVideoLen;
         memset(audioData, 0x00, partition->dataBufferAudioLen);
         return true;
     }
-    if (MediaSourceType::VIDEO == mediaSourceType)
+    if (MediaSourceType::SUBTITLE == mediaSourceType)
     {
-        std::uint8_t *videoData = partitionDataPtr;
```

Comment:
Reminder to update shm documentation https://wiki.rdkcentral.com/display/ASP/Rialto+Playback+Design